src/main/resources/application-dev.yml

```
@@ -0,0 +1,4 @@
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: update
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)